site/class-1-play/NataliaHartgers.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="600" height="600">

```

Comment:
Remove this file (go to this file on your branch on your fork, and click on the trash icon in the top-right)